packages/ui/cypress/e2e/case-details/view-case-details/view-notes.cy.ts

```diff
@@ -133,18 +133,6 @@ describe("View notes", () => {
     cy.contains("Case has no notes.")
   })
 
-  it("Should not display the notes text area if the case is locked to another users", () => {
-    cy.task("insertCourtCasesWithNotesAndLock", {
-      caseNotes: [[]],
-      force: "01"
-    })
-
-    loginAndGoToNotes()
-    cy.contains("Case has no notes.")
-    cy.get("textarea[name=noteText]").should("not.exist")
-    cy.get("#add-note-button").should("not.exist")
-  })
-
```

Comment:
Should this test now be doing the opposite? We want to see the button now. Does another test cover this?

Reply:
Also, that's a very specific cypress task, can it be removed if nothing else is using that?

Reply:
I'm going to remove any notion of the idea of "locking" in the tests as it's no longer relevant, I'll add another test for the adding notes to resolved cases requirement